Dockerfile

```diff
@@ -5,33 +5,17 @@
 # Both ubuntu and eclipse-temurin are pinned via digest and not by a strict version tag, as Renovate
 # has trouble with custom versioning schemes
 ARG BASE_IMAGE="ubuntu:jammy"
-ARG BASE_DIGEST_AMD64="sha256:b492494d8e0113c4ad3fe4528a4b5ff89faa5331f7d52c5c138196f69ce176a6"
-ARG BASE_DIGEST_ARM64="sha256:94d12db896d07af18a04319f1023edd091629f558dcc29f84208a7cf5ca040ec"
+ARG BASE_DIGEST="sha256:aabed3296a3d45cede1dc866a24476c4d7e093aa806263c27ddaadbdce3c1054"
 ARG JDK_IMAGE="eclipse-temurin:17-jdk-jammy"
-ARG JDK_DIGEST_AMD64="sha256:0dfd9830fdf7851876aa6f830c9050eedcc9a25921eff64b03259589d3083d9f"
-ARG JDK_DIGEST_ARM64="sha256:0e0cd6c795872211420e4a1cf9468ad607543ef7c14d0c12d01eb93c7ccf1d15"
+ARG JDK_DIGEST="sha256:cf2e1d79e3d430b5277a1ff04cedb521706a053a2adc17b384e9a9bb4b1e79ee"
 
 # set to "build" to build zeebe from scratch instead of using a distball
 ARG DIST="distball"
 
-### AMD64 base image ###
-# BASE_DIGEST_AMD64 is defined at the top of the Dockerfile
-# hadolint ignore=DL3006
-FROM ${BASE_IMAGE}@${BASE_DIGEST_AMD64} as base-amd64
-ARG BASE_DIGEST_AMD64
-ARG BASE_DIGEST="${BASE_DIGEST_AMD64}"
-
-### ARM64 base image ##
-# BASE_DIGEST_ARM64 is defined at the top of the Dockerfile
-# hadolint ignore=DL3006
-FROM ${BASE_IMAGE}@${BASE_DIGEST_ARM64} as base-arm64
-ARG BASE_DIGEST_ARM64
-ARG BASE_DIGEST="${BASE_DIGEST_ARM64}"
-
-### Base image ##
+### Base image ###
 # All package installation, updates, etc., anything with APT should be done here in a single step
 # hadolint ignore=DL3006
-FROM base-${TARGETARCH} as base
+FROM ${BASE_IMAGE}@${BASE_DIGEST} as base
```

Comment:
:upside_down_face: Now we don't even need the variables, right? We can just inline both, same for the JDK variables.

Reply:
... :man_facepalming:

yes :smile:

Reply:
That'll make finding the digest harder in the verify script :smile:

Reply:
Ah true. Then I'm fine leaving as is :shrug: